wsdl/ver10/schema/onvif.xsd

```diff
@@ -1850,6 +1912,16 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 					<xs:documentation>Minimum and maximum level range supported for this Output.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="EQPresetScheduleSupport" type="xs:boolean" minOccurs="0">
```

Comment:
If it is true does it meant that Schedule Service shall be supported by a DUT?

Reply:
Yes, when EQPresetScheduleSupport is true, the schedule service is supported by the DUT and is also available in the EQPreset feature.